bigquery-storage/api/BigQueryStorage.Samples/AppendRowsPending.cs

```diff
@@ -0,0 +1,145 @@
+        // In Production scenario, these records would be passed to method.
+        List<CustomerRecord> records = new List<CustomerRecord>
+        {
+            new CustomerRecord { CustomerNumber = 1, CustomerName = "Alice" },
+            new CustomerRecord { CustomerNumber = 2, CustomerName = "Bob" }
+        };
+
+        // Create a batch of row data by appending serialized bytes to the
+        // SerializedRows repeated field.
+        ProtoData protoData = new ProtoData
+        {
+            // This is a TEMPORARY WORKAROUND via extension method using reflection until
+            // https://github.com/protocolbuffers/protobuf/issues/9425 is addressed and released.
+            WriterSchema = new ProtoSchema { ProtoDescriptor = CustomerRecord.Descriptor.ToProto() },
+            Rows = new ProtoRows { SerializedRows = { records.Select(r => r.ToByteString()) } }
+        };
+
+        // Initialize the append row request.
+        AppendRowsRequest appendRowRequest = new AppendRowsRequest
+        {
+            WriteStreamAsWriteStreamName = stream.WriteStreamName,
+            ProtoRows = protoData
+        };
+
+        // Stream a request to the server.
+        await rowAppender.WriteAsync(appendRowRequest);
+
+        // Append a second batch of data.
+        protoData = new ProtoData
+        {
+            Rows = new ProtoRows { SerializedRows = { new CustomerRecord { CustomerNumber = 3, CustomerName = "Charles" }.ToByteString() } }
+        };
+
+        // Since this is the second request, you only need to include the row data.
+        // The name of the stream and protocol buffers DESCRIPTOR is only needed in
+        // the first request.
+        appendRowRequest = new AppendRowsRequest
+        {
+            // If Offset is not present, the write is performed at the current end of stream.
+            ProtoRows = protoData
+        };
+
+        await rowAppender.WriteAsync(appendRowRequest);
+
+        // Complete writing requests to the stream.
+        await rowAppender.WriteCompleteAsync();
+
+        // Await the handler. This will complete once all server responses have been processed.
+        await appendResultsHandlerTask;
+
+        // A PENDING type stream must be "finalized" before being committed. No new
+        // records can be written to the stream after this method has been called.
+        await bigQueryWriteClient.FinalizeWriteStreamAsync(stream.Name);
+        BatchCommitWriteStreamsRequest batchCommitWriteStreamsRequest = new BatchCommitWriteStreamsRequest
+        {
+            Parent = table.ToString(),
+            WriteStreams = { stream.Name }
+        };
+
+        BatchCommitWriteStreamsResponse batchCommitWriteStreamsResponse =
+            await bigQueryWriteClient.BatchCommitWriteStreamsAsync(batchCommitWriteStreamsRequest);
+        // Commit Time is set only if batch commit is successful.
+        if (batchCommitWriteStreamsResponse.StreamErrors!.Count > 0)
```

Comment:
I know it was my suggestion before, but it should have been:
```suggestion
        if (batchCommitWriteStreamsResponse.StreamErrors?.Count > 0)
```

Reply:
Done.